server/migrations/1584377551030-AddCancelledColumnToRsvp.ts

```diff
@@ -0,0 +1,13 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddCancelledColumnToRsvp1584377551030 implements MigrationInterface {
+  name = 'AddCancelledColumnToRsvp1584377551030';
+
+  public async up(queryRunner: QueryRunner): Promise<any> {
+    await queryRunner.query(`ALTER TABLE "rsvps" ADD "cancelled" BOOLEAN NOT NULL`, undefined);
```

Comment:
I know cancelled vs canceled is a location-varying thing, but I think we're already using "canceled" on the _events_ table, so I'd say to keep it consistent for that reason.

Reply:
Of course, this would apply to all the places using "cancelled".